mlflow/ml-package-versions.yml

```diff
@@ -85,6 +85,7 @@ tensorflow:
       ">= 0.0.0": ["scikit-learn", "pyspark", "pyarrow", "transformers"]
       "< 2.6.0": ["protobuf<4.0.0"]
       "< 2.7.0": ["pandas==1.3.5"]
+      ">= 2.7.0": ["pandas<2.0"]
```

Comment:
Pinned pandas as a temporary fix. I'll fix tests next week so that we can unpin pandas.